src/mysql.go

```diff
@@ -39,8 +41,25 @@ func generateDSN(args argumentList) string {
 var args argumentList
 
 func main() {
-	i, err := integration.New(integrationName, integrationVersion, integration.Args(&args))
+
+	var i *integration.Integration
+	var err error
+	cachePath := os.Getenv("NRIA_CACHE_PATH")
+	if cachePath == "" {
```

Comment:
In my integration I moved this code into a function which looks clearer for me:
https://github.com/newrelic/nri-redis/pull/21/files#diff-118b55cb154646993f79f78a6b5efe3eR92

Reply:
Thanks. I'll use it in a new PR